torch/lib/c10d/test/CUDATest.hpp

```diff
@@ -5,9 +5,15 @@
 namespace c10d {
 namespace test {
 
-void cudaSleep(at::cuda::CUDAStream& stream, uint64_t clocks);
+#ifdef _WIN32
+#define EXPORT_API extern "C" __declspec(dllexport)
```

Comment:
Why does it need to be extern "C"
And why it needs to be exported only on Windows platform?